### ISSUE TITLE
training log file and curves

### ISSUE THREAD

**Issue by Ivamcoder** [edgarschnfld/CADA-VAE-PyTorch#9]
Hi @edgarschnfld,

Thanks for your paper and code, I was wondering do you have log file  and training curves,
I am interested in reconstruction loss and kld losses and Wasserstein distance curves while training, and their coefficient,

I would appreciate if you share your trained log files to us. It would be so useful,

**Comment by edgarschnfld** [comment]
Hey, I don't have log files or training curves, unfortunately. However, running the code takes about 5-10 minutes, so it should be easy to write the loss values to a file and compare. Sorry I couldn't help, but feel welcome to write me if you have more questions.
Best,
Edgar
